### ISSUE TITLE
can your provided the performance in visda2017

### ISSUE THREAD

**Issue by valencebond** [cuishuhao/BNM#6]
there is visda-2017 in DA/data/, but not training script in README file, does this code support training in the visda database?

**Comment by cuishuhao** [comment]
I ran visda2017 and obtained results about 72%. 
To train visda2017, simply changing the learning rate and class number is ok.
You can refer to https://github.com/thuml/CDAN/blob/master/pytorch/train_image.py#L266

**Comment by cuishuhao** [comment]
@valencebond Results of visda-2017 are also supported in https://github.com/tim-learn/ATDOC/blob/main/demo_uda.py#L228
